### PR TITLE
Add Vanilla spoofing

### PR DESCRIPTION
**Pull request by U5B**
Basically spoofs your client as Vanilla

Also `\7vanilla` is the exact thing needed to connect to hypixel properly since just `vanilla` wouldn't work.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,13 +29,18 @@ const proxy = new Proxy({
...
+    if (data.channel === 'badlion:mods' || data.channel === 'FML|HS' || data.channel === 'FML') return
     else if (data.channel === 'MC|Brand') data.data = Buffer.from('<XeBungee (git:XeBungee-Bootstrap:1.16-R0.5-SNAPSHOT:a2e1df4)') // hack to re-enable freelook on lunar
   }
   toClient.write(meta.name, data)
 })
 
 proxy.on('outgoing', (data, meta, toClient, toServer) => {
+  if (meta.name === 'custom_payload') {
+    if (data.channel === 'MC|Brand') {
+      data.data = Buffer.from('\7vanilla')
```
Comment (u9g) [inline index.js]: How did you get this string? I just used pakkit and got this: 
image
which isn't equal to that

Comment (U5B) [reply index.js]: I compared what the vanilla 1.8.9 client sends to the server and mimicked that. The \7 is the varint prefix, and the rest spells out "vanilla".

Comment (U5B) [reply index.js]: https://wiki.vg/index.php?title=Plugin_channels#:~:text=this%20is%20%22-,vanilla,-%22%20(which

File: index.js [hunk 2]
```diff
@@ -29,13 +29,18 @@ const proxy = new Proxy({
 
 proxy.on('incoming', (data, meta, toClient, toServer) => {
   if (meta.name === 'custom_payload') {
-    if (data.channel === 'badlion:mods') return
+    if (data.channel === 'badlion:mods' || data.channel === 'FML|HS' || data.channel === 'FML') return
```
Comment (u9g) [inline index.js:32]: should we include all the aliases https://i.imgur.com/J5ROPYa.png

Comment (U5B) [reply index.js:32]: if you intend to support 1.13+
that is used in FML2

**Comment by u9g** [comment]
Thanks for contributing!

**Pull request merged**